tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: install dependencies
+  apt: name={{ item }} state=present
+  with_items: "{{ BACKUP_SWIFT_CONTAINER_DEPS }}"
+
+- name: create backup dir
+  file: name="{{ BACKUP_SWIFT_LOCAL_DIR }}" state=directory mode="0700"
+
+- name: save backup script
+  template:
+    src: backup-swift.sh
+    dest: "{{ BACKUP_SWIFT_DEST }}"
+    owner: "root"
+    mode: "0700"
```

Comment:
There are parts of this role that duplicate steps from https://github.com/open-craft/ansible-opencraft-tarsnap/pull/1 such as uploading the backup script, and executing it by cron - which can be factorized by making this role depend on https://github.com/open-craft/ansible-opencraft-tarsnap/pull/1  and set the backup script as the pre backup script.

Also, do I understand correctly that we have two different approaches for backups, some using tarsnapper, some not like this one? I'd like to chose one approach and use it consistently.

Also, similarly to having restrictive default rules for ufw, I'd like this to be part of the common roles, and set in a way where the playbook fails if the config for tarsnap (or tarsnaper) *and* DMS isn't explicitly set on a given host - to be sure we always remember to set up a backup.